modules/apps/object/object-test/src/testFunctional/paths/ObjectPortlet.path

```diff
@@ -225,6 +225,11 @@
 		<td>//button[contains(@class,'btn-sorting') and contains(.,'${key_orderColumn}')]//*[contains(@class,'lexicon-icon-order')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>RELATIONSHIP_ENTRY_READ_ONLY</td>
+		<td>//div[contains(@class,'ddm-field')]//input[contains(., readonly) and contains(@value, '${key_entry}')]</td>
```

Comment:
When you want to check if a HTML element has a certain attribute, as in `contains(., readonly)`,  you can directly add `@readonly`. For exemple, [here](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TextInput.path#L505).